themes/default/content/learn/building-with-pulumi/stack-references/index.md

```diff
@@ -127,6 +137,47 @@ export then grabs the `url` output from the other stack.
 
 {{% /choosable %}}
 
+{{% choosable language go %}}
+
+```go
+package main
+
+import (
+	"fmt"
+
+	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
+	"github.com/pulumi/pulumi/sdk/v3/go/pulumi/config"
+)
+
+func main() {
+	pulumi.Run(func(ctx *pulumi.Context) error {
+		cfg := config.New(ctx, "")
+		stack := ctx.Stack()
+		org := cfg.Require("org")
+
+		stackRef, err := pulumi.NewStackReference(ctx, fmt.Sprintf("%v/my-first-app/%v", org, stack), nil)
+		if err != nil {
+			return err
+		}
+
+		ctx.Export("shopUrl", stackRef.GetOutput(pulumi.String("url")))
+		return nil
+	})
+}
+```
+
+You'll also need to run `go mod tidy` in the `my-second-app` directory.
+
+The `org` environment variable is new, as is the `stackRef` declaration. That
+declaration sets up an instance of the `StackReference` class, which needs the
+fully qualified name of the stack as an input. Here, the `org` is the
+organization associated with your account, the `my-first-app` is the name of the
```

Comment:
Also not this PR, but it's a little awkward that we repeat "the organization associated with your account" here:

<img width="881" alt="image" src="https://user-images.githubusercontent.com/274700/211641364-31b0d5d5-dae7-4556-88a7-da3393e4a372.png">